day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,65 @@
+def in_words(int)
+  numbers_to_name = {
+      1000000 => "million",
+      1000 => "thousand",
+      100 => "hundred",
+      90 => "ninety",
+      80 => "eighty",
+      70 => "seventy",
+      60 => "sixty",
+      50 => "fifty",
+      40 => "forty",
+      30 => "thirty",
+      20 => "twenty",
+      19=>"nineteen",
```

Comment:
Consistency is key. Notice how this line and below looks different than above? I like the look of the above lines more for readability.